openassessment/xblock/static/js/src/lms/oa_base.js

```diff
@@ -69,6 +69,13 @@ OpenAssessment.BaseView.prototype = {
     },
 
     /**
+     * Clear the text in the Aria live region.
+     */
+    srClear: function() {
+        $('.sr.reader-feedback').html('');
```

Comment:
I would define .sr.reader-feedback as a class variable, since you use it in a couple of methods.